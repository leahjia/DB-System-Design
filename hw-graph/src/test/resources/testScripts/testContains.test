# University Map Contains Node testing file.
# Tests whether a Node is contained in the university map

# checks whether building A exists in UnivMap
contains Node A

# checks whether building B exists in UnivMap
contains Node B