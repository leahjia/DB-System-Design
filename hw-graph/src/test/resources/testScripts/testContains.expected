# University Map Contains Node testing file.
# Tests whether a Node is contained in the university map

# checks whether building A exists in UnivMap
map1 contains: A

# checks whether building B exists in UnivMap
map1 contains: B
